### PR TITLE
Returning the old laser rifles, energy pistols, carbines and much more

### PR DESCRIPTION
**Pull request by Alberyk**
In this update:

- Returning old e-rifles from old code.
- Changing regular energy gun to energy carbine
- Adding back the energy pistol
- Buffing a bit the heavy asset protection suit, and fixing more loadout related issues with the nt and syndicate assault teams
- Reworked a bit the syndicate commando and deathsquad loadouts
- Returning shotgun boxes, that works like speed loaders, from old code, now with unique sprites
- Stun batons now emit light

### CODE REVIEW

File: code/game/antagonist/outsider/commando.dm [hunk 1]
```diff
@@ -25,12 +25,12 @@ var/datum/antagonist/deathsquad/mercenary/commandos
 	player.equip_to_slot_or_del(new /obj/item/clothing/gloves/swat(player), slot_gloves)
 	player.equip_to_slot_or_del(new /obj/item/clothing/glasses/thermal(player), slot_glasses)
 	player.equip_to_slot_or_del(new /obj/item/clothing/mask/gas/syndicate(player), slot_wear_mask)
-	player.equip_to_slot_or_del(new /obj/item/weapon/storage/box(player), slot_in_backpack)
-	player.equip_to_slot_or_del(new /obj/item/ammo_magazine/c45(player), slot_in_backpack)
+	player.equip_to_slot_or_del(new /obj/item/ammo_magazine/c45(player), slot_l_store)
+	player.equip_to_slot_or_del(new /obj/item/weapon/melee/energy/sword(player), slot_r_store)
```
Comment (Bedshaped) [inline code/game/antagonist/outsider/commando.dm]: adding e-swords to commandos should probably be changelogged too since it would buff them.

Comment (Alberyk) [inline code/game/antagonist/outsider/commando.dm]: Hardly, since, they are just an admin bus thing, so, don't really matter to most player base.

**Comment by skull132** [comment]
Does this modify the armoury random item spawners, or are you leaving those as laser guns?

**Comment by Alberyk** [comment]
It does, I changed it in the random.dm

**Pull request merged**